engine/updategraph/src/main/java/io/deephaven/engine/util/systemicmarking/SystemicObjectTracker.java

```diff
@@ -34,14 +34,23 @@ public static boolean isSystemicThread() {
     }
 
     /**
-     * Marks the current thread as systemically important, this is a permanent change.
+     * Marks the current thread as systemically important. This can be changed with {@link #markThreadNotSystemic()}
      */
     public static void markThreadSystemic() {
         if (SYSTEMIC_OBJECT_MARKING_ENABLED) {
             SYSTEMIC_CREATION_THREAD.set(true);
         }
     }
 
+    /**
+     * Marks the current thread as systemically important. This can be changed with {@link #markThreadSystemic()} ()}
+     */
+    public static void markThreadNotSystemic() {
```

Comment:
Should we also have `executeNonSystemically`? That pattern is much easier to use correctly.

Reply:
Never mind, I see that `executionSystemic` has a boolean arg.

Reply:
The other method has a boolean though -- you can tell it true or false for systemic.